lib/model/channel.dart

```diff
@@ -78,6 +84,29 @@ mixin ChannelStore on UserStore {
     };
   }
 
+  /// Fetch topics in a channel from the server, only if they're not fetched yet.
+  ///
+  /// The results from the last successful fetch
+  /// can be retrieved with [getChannelTopics].
+  Future<void> fetchTopics(int channelId);
```

Comment:
Seems like we'd want to call this `fetchChannelTopics`, to be more parallel to `getChannelTopics`.